src/components/scenes/ManageTokensScene.js

```diff
@@ -260,15 +260,12 @@ class ManageTokensSceneComponent extends React.Component<Props, State> {
               />
             </View>
             <SceneFooter style={styles.buttonsArea} underline>
```

Comment:
Um, actually, the `SceneFooter` component is garbage. It's only used on this one scene, it takes explicit CSS (big no-no), and even uses `Dimensions.get` inside it's CSS!

Can we create a `DividerLine` component that just renders the line, and then put that on top of a normal `View` to lay out the buttons?

The `DividerLine` would have `marginRight: -theme.rem(1)` to stick out over the right side, so it can be positioned inside a `padding: theme.rem(1)` container and look perfect.